admin/listPurchases/listView.html

```diff
@@ -199,6 +199,21 @@
     }
 }
 
+function describeEntry(entry){
+    const preOrderTag = entry.isPreorder ? 'PRE:<br> ' : '';
+    const describedItems = entry.items
+        .map((item) => describeItem(item))
+        .join("<br>");
+    return `${preOrderTag}${describedItems}`;
+}
+
+function describeItem(item){
```

Comment:
Missing space between `)` and `{` for consistency with other function definitions in this code base. Same with `describeEntry` above